website/docs/error-reference.md

```diff
@@ -790,6 +790,34 @@ class A::B < PackageSpec
 end
 ```
 
+Additional signatures of error 3721 include:
+
+- A package exporting a constant only defined in .rbi files. RBI files are shims
+  to enable typechecking in places where Ruby metaprogramming prevents Sorbet
+  from statically interpreting the behavior of a class or module. Generally,
+  these files declare additional methods on classes defined in Ruby source
+  files, and should not define any new constants. However, there are some rare
+  exceptions where these files can define net-new constants. For these cases, we
+  enforce that these constants cannot be exported.
+- A package exporting an enum value:
+
+```ruby
+module MyPackage
+  class A < T::Enum
+    enums do
+      Val1 = new
+      Val2 = new
+    end
+  end
+end
```

Comment:
Open to turning each signature into a new error code.

Reply:
Yeah I think we want to make a new code for this. It's not obvious to me from skimming whether I care about this section of the 3721 docs or the previous section, so I think that explicitly being able to drop people into the relevant section will help.

Reply:
I can make this change (breaking out the error codes) in a different PR.